mobile/mobile_46.asm

```diff
@@ -956,7 +956,7 @@ Function11878d:
 .asm_118803
 	ld a, $d3
 
-Function118805:
+Mobile46_SetErrorCode:
```

Comment:
Just `Mobile_`

Reply:
I used Mobile46 since there'll be equivalent functions in other bank

Reply:
Alright then, though it'd be better if you could name them something else. Are they identical or are they serving for different purposes or deal with different data/codes?
If not this name is ok.

Reply:
If its a problem i can just undo that change, the only reason i named it is so at call sites its apparent that branch is handling an error

Having another look i actually cant find an equivalent function, its unique to bank 46 and other banks just have its code inlined, but given that it Is unique to that bank a generic name like "Mobile_" could be misleading

Reply:
Since the value it's setting is `wMobileErrorCodeBuffer`, how about `SetMobileErrorCode`? It's fine if the function is unique to this bank, there's no implication that it has to be usable in every bank (and if it were, it could always be `farcall`ed, not that any hacks are editing the mobile system anyway.)